### ISSUE TITLE
Bug on call transfer to the external world

### ISSUE THREAD

**Issue by glacroix** [irontec/ivozprovider#1010]
Hello,

I could not make call transfer to an external number. After some debugging, it turns out that the external number was not properly transformed so it was rejected by kamailio@trunks.

The bug is due to a tiny typo in kamailio.cfg of kamailio@users : 
```
route[ADAPT_REFERTO] {
# Only in within-dialog REFER requests from subscribers
    if(!is_method("REFER")) return;
    if($dlg_var(type ) != 'vpbx') return;

    if (search_hf("Refer-To", "Replaces", "a")) {
```
There is one extra space after 'type', so the route  returned immediately. So it should be instead : 
`if($dlg_var(type) != 'vpbx') return;`

And then it works :) 

Hope it helps !

Bye,
Guillaume

**Comment by Kaian** [comment]
Hi @glacroix 

Thanks for the bug report and patch!

Are you willing to Pull Request this change?

Regards!

**Comment by glacroix** [comment]
Hi @Kaian,
Thanks ! Sure, I'm pretty new to Github, how can I do that :) ?

**Comment by Kaian** [comment]
A change that simple can be easily done from Github web.

 - [Fork the project](https://help.github.com/en/articles/fork-a-repo) (press the fork button in the upper right corner)
 - In your forked project, go to the file and press the pencil icon to [edit the file](https://help.github.com/en/articles/editing-files-in-your-repository)
 - In the bottom part of the editor set the commit message (for example _trunks: fix typo in transferred external calls_) and select create a new branch from this commit. 
- Then go to [our project](https://github.com/irontec/ivozprovider/) and a button will appear to compare your branch and [create a new Pull request](https://help.github.com/en/articles/creating-a-pull-request-from-a-fork)
- Enter a description (something like the text of this issue) and submit your PR

If this process is too complicated for you, dont worry, I will do it for you. Otherwise you will have a new knowledge that can reuse in the future to submit changes to the project

Regards!

**Comment by glacroix** [comment]
Got it and done !

Thanks a lot for this new knowledge ;)

**Comment by Kaian** [comment]
Fixed in #1013

Good, I rewrote the commit title but has been merged. Also, next time, select `bleeding` as destination branch.

Regards!!
